libkbfs/folder_branch_ops.go

```diff
@@ -4704,6 +4704,11 @@ func (fbo *folderBranchOps) applyMDUpdatesLocked(ctx context.Context,
 				return err
 			}
 		}
+		if rmd.IsRekeySet() {
```

Comment:
In slack I mentioned we should only do this if the update is from some other device, to make sure our own updates don't cause issues.  Maybe the revision check above is good enough to prevent this, but I just want to make sure you thought about it.